docs/code-snippets/outlook-snippets.yaml

```diff
@@ -119,6 +119,151 @@ Office.Item.loadCustomPropertiesAsync:
 
     function saveCallback(asyncResult) {
     }
+Office.Mailbox.addHandlerAsync:
+  - |-
+    Office.initialize = function (reason) {
+        $(document).ready(function () {
+            Office.context.mailbox.addHandlerAsync(Office.EventType.ItemChanged, loadNewItem, function (result) {
+            if (result.status === Office.AsyncResultStatus.Failed) {
+                    // Handle error
+                }
+            });
+        });
+    };
+
+    function loadNewItem(eventArgs) {
+        // Load the properties of the newly selected item
+        loadProps(Office.context.mailbox.item);
+    };
+Office.Mailbox.convertToEwsId:
+  - |-
+    // Get an item's ID from a REST API
+    var restId = 'AAMkAGVlOTZjNTM3LW...';
+
+    // Treat restId as coming from the v2.0 version of the Outlook Mail API
+    var ewsId = Office.context.mailbox.convertToEwsId(restId, Office.MailboxEnums.RestVersion.v2_0);
+Office.Mailbox.convertToRestId:
+  - |-
+    // Get the currently selected item's ID
+    var ewsId = Office.context.mailbox.item.itemId;
+
+    // Convert to a REST ID for the v2.0 version of the Outlook Mail API
+    var restId = Office.context.mailbox.convertToRestId(ewsId, Office.MailboxEnums.RestVersion.v2_0);
+Office.Mailbox.displayAppointmentForm:
+  - |-
+    Office.context.mailbox.displayAppointmentForm(appointmentId);
```

Comment:
What's the value of these one-line snippets? Isn't this the same as the function signature?